### PR TITLE
flight-rule to stash multiple files at once

### PR DESCRIPTION
**Pull request by flexdinesh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -902,15 +902,21 @@ If you also want to stash untracked files, use `-u` option.
...
 To stash only one file from your working directory
 
 ```sh
 $ git stash push working-directory-path/filename.ext
 ```
 
+To stash multiple files from your working directory
+
+```sh
+$ git stash push working-directory-path/filename1.ext <space> working-directory-path/filename2.ext
```
Comment (RichardLitt) [inline README.md]: What does the `<space>` signify?

Comment (RichardLitt) [reply README.md]: I don't think that's useful - a space itself would do this. Can we remove it?

Comment (eamanu) [reply README.md]: something like this:  

$ git stash push working-directory-path/filename1.ext working-directory-path/filename2.ext ?

Comment (RichardLitt) [reply README.md]: Yep!

Comment (eamanu) [reply README.md]: Fix it

**Pull request merged**